src/hooks/hooks.ts

```diff
@@ -0,0 +1,8 @@
+import { useDispatch, useSelector } from "react-redux";
+import type { TypedUseSelectorHook } from "react-redux";
+import type { RootState, AppDispatch } from "../store/store";
+
+export const useAppDispatch: () => AppDispatch = useDispatch;
```

Comment:
This is useless, you can use `useDispatch` directly.

Reply:
I found this in the 'redux toolkit for typescript' documentation:
https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-dispatch-type